components/CreateEventForm/CreateEventForm.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { StyleSheet, View, Button, Text, TextInput} from 'react-native';
+
+export default function CreateEventForm() {
+  return (
+    <View style={styles.container}>
+      <View style={styles.closeButton}><Button title='X'/></View>
+      <Text style={styles.text}>CREATE A NEW EVENT FORM!</Text>
+      <TextInput style={styles.input} placeholder='Event Name...'/>
+      <TextInput style={styles.input} placeholder='Location...'/>
+      <TextInput style={styles.input} placeholder='Start Time...'/>
+      <TextInput style={styles.input} placeholder='End Time...'/>
+      <TextInput style={styles.input} placeholder='Total Players Required...'/>
+      <TextInput style={styles.input} placeholder='Players Curretly Attending...'/>
+      <TextInput style={styles.input} placeholder='Skill Level...'/>
+      <TextInput style={styles.input} placeholder='Equipment Reqired...'/>
```

Comment:
This will probably need to be a dropdown in the future

Reply:
Good idea, I was thinking this would probably need to a drop down menu or something similar.